packages/cli/src/commands/org/config/preview.ts

```diff
@@ -41,21 +41,16 @@ export default class Preview extends Command {
     const {flags} = this.parse(Preview);
     const project = new Project(flags.projectPath);
     const pathToZip = await project.compressResources();
-
-    const factory = new SnapshotFactory();
-    const snapshot = await factory.createFromZip(
-      pathToZip,
-      'cli-preview-snapshot'
-    );
+    const targetOrg = await this.getTargetOrg();
 
     cli.action.start('Creating snapshot');
 
-    const targetOrg = await this.getTargetOrg();
-    await snapshot.push(targetOrg, 'cli-preview-snapshot');
+    const factory = new SnapshotFactory();
```

Comment:
would be impacted by my comment about the snapshotFactory, stateless class, no need for `new` ;)

Reply:
Correct. I will be fix once I merge the corrections from CDX-353